brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift

```diff
@@ -0,0 +1,259 @@
+// 
+//  SwapCurrencyView.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import UIKit
+
+class SwapCurrencyView: UIView {
+    var didChangeFiatAmount: ((SwapMainModels.Amounts.CurrencyData?) -> Void)?
+    var didChangeCryptoAmount: ((SwapMainModels.Amounts.CurrencyData?) -> Void)?
+    
+    private lazy var currencyContainerView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    private lazy var currencyHeaderStackView: UIStackView = {
+        let view = UIStackView()
+        view.axis = .horizontal
+        view.distribution = .equalSpacing
+        return view
+    }()
+    
+    lazy var iHaveLabel: FELabel = {
+        let view = FELabel()
+        view.text = "I have 100 BSV"
+        view.font = Fonts.caption
+        view.textColor = LightColors.Icons.one
+        view.textAlignment = .left
+        return view
+    }()
+    
+    lazy var fiatTitleStackView: UIStackView = {
+        let view = UIStackView()
+        view.axis = .horizontal
+        view.distribution = .fill
+        view.spacing = Margins.extraSmall.rawValue
+        return view
+    }()
+    
+    lazy var fiatAmountField: UITextField = {
+        let view = UITextField()
+        view.textColor = LightColors.Icons.one
+        view.font = Fonts.Subtitle.two
+        view.attributedPlaceholder = NSAttributedString(
+            string: "0.00",
+            attributes: [NSAttributedString.Key.foregroundColor: view.textColor ?? UIColor.black,
```

Comment:
if u guard against values, then u dont have 2 have fallbacks to nonstandard colors/fonts
(values should again come from the config)